.github/workflows/check-pr.yml

```diff
@@ -49,9 +50,9 @@ jobs:
     - name: Check breaking change label
       id: check_breaking_change
       run: |
-        title="${{ github.event.pull_request.title }}"
         pattern='^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\(\w+\))?!:'
-        if echo "$title" | grep -qE "$pattern"; then
+        # Check if pattern matches
+        if echo "${{ github.event.pull_request.title }}" | grep -qE "$pattern"; then
```

Comment:
We could probably do something like this here:

```
if [[ "${{ github.event.pull_request.title }}" =~ $pattern ]]; then
```

But I'm too late for the review, can you guys wait a bit before merge?

Reply:
Sorry about that! Next time I'll wait longer for merges.